docs/VnVPlan/VnVPlan.tex

```diff
@@ -549,8 +549,318 @@ \subsection{Tests for Nonfunctional Requirements}
+
+Type: Manual, Front End Review
+					
+Initial State: Application on home page, ready for use
+					
+Input/Condition: Tester engagement
+					
+Output/Result: List of all identified inconsistencies that are found that do not adhere to SRS defined requirements.
+					
+How test will be performed: 
+\begin{enumerate}
+  \item A developer will review the code to make sure the look and feels requirements were implemented
+  \item They will then look at the application as well to ensure that the implementation was made correctly
+\end{enumerate}
+					
+\item{NFR-UI2}
+
+NFR: CR-SC3
+
+Type: Manual, Accessibility Check
+					
+Initial State: Application on home page, ready for use
+					
+Input: Tester engagement
+					
+Output: List of missing accessibility considerations that have been overlooked
+					
+How test will be performed: 
+\begin{enumerate}
+  \item The developer will review the code and application to ensure that accessibility considerations have been implemented
+  \item They will verify that all images have alt text
+  \item The developer will note down any missing implementations of accessibility requirements
+\end{enumerate}
+
+\end{enumerate}
 
 \subsubsection{Usability and Humanity Requirements}
+These requirements are intended to build a foundation for the user's experience
+on the application. Thus, the only real way to get feedback the user experience
+is to have someone use the application and provide feedback.
+
+\begin{description}
+  \item[User Test Demo:] We will engage a user to test the application, bringing
+  them through a new user process, and ask for feedback on specific areas
+  relevant to the requirements that were set.
+\end{description}
+
+
+\begin{enumerate}
+
+  \item{NFR-UH1}
+  
+  NFR: UH-E1, UH-E3, UH-L1, UH-UP1, UH-UP2, OE-P1
+  
+  Type: Manual, User Test Demo
+            
+  Initial State: Application on home page, ready for use
+            
+  Input/Condition: User engagement
+            
+  Output/Result: Subjective feedback for how well application scores on different requirements relating to user experience
+            
+  How test will be performed: 
+  \begin{enumerate}
+    \item A user will use the application, going through the onboarding process
+    and doing a demo of the application
+    \item After a full demo, the user will be asked to provide feedback on
+    different aspects of usability, such as ease of use, app clarity, and prompt
+    politeness.
+  \end{enumerate}
+            
+  \item{NFR-UH2}
+  
+  NFR: UH-E2
+  
+  Type: Manual
+            
+  Initial State: Application on home page, ready for use
+            
+  Input: Tester engagement
+            
+  Output: A count for the number of clicks required to navigate to different functionality
+            
+  How test will be performed: 
+  \begin{enumerate}
+    \item A list of commonly used functions will be created (System is not large, therefore this can be done manually)
```

Comment:
What are the commonly used functions? We need to list them out.
Here's what I had in mind (only really these two matter):
- Make a submission
- View a generated report

Reply:
I literally said a list of commonly used functions will be created

Reply:
You can't just say the list will be created, we have to actually have the list. That's the point of the test.

Reply:
Addressed in 917cc17c45140145e766d64cb16526099f24393d

Reply:
As in, the list will be made when we do the actual tests